db/main/migrate/20161124171800_create_features_table.rb

```diff
@@ -0,0 +1,16 @@
+class CreateFeaturesTable < ActiveRecord::Migration
+  def change
+    create_table :features do |t|
+      t.string :name
+      t.string :feedback_url
+      t.references :owner, polymorphic: true
+      t.boolean :public
+      t.boolean :modifiable
+      t.boolean :active
+      t.timestamp :expires_on
+      t.timestamps
```

Comment:
as per convo w/ @joecorcoran we should add a json (or the type for postgres 9.3/4 that we use) column for additional settings (example: they have limit max builds feature turned on and it's set to 11)